src/generator/gen_unicode.mli

```diff
@@ -0,0 +1 @@
+(* empty on purpose *)
```

Comment:
You can use `empty_module_interface_if_absent` from `dune` for that: https://dune.readthedocs.io/en/stable/dune-files.html#executable